packages/dev/sharedUiComponents/src/fluent/primitives/spinButton.tsx

```diff
@@ -69,8 +70,10 @@ export const SpinButton: FunctionComponent<SpinButtonProps> = (props) => {
     };
 
     const id = useId("spin-button");
+    const mergedClassName = mergeClasses(classes.input, !validateValue(value) ? classes.invalid : "", props.className);

```

Comment:
I think you can also just pass null or undefined to mergeClasses if I recall (rather than empty string). Also, how heavy is `mergeClasses`? Should this be in a `useMemo`?